### DISCUSSION TITLE
gpt-4不能用

### DISCUSSION THREAD

**Discussion by eric-cleath** [nextai-translator/nextai-translator#418] [Q&A]
<img width="387" alt="SCR-20230321-ltjs" src="https://user-images.githubusercontent.com/60673256/226687254-4ee04b53-5c3e-4bf9-84a9-3a3b6e15844d.png">

**Comment by FunkyTo9** [comment]
需要添加选项 text-davinci-002, text-davinci-003 , code-davinci-002

**Reply by eric-cleath** [reply]
请问怎么添加呢，谢谢。

**Reply by wongsiuhoi** [reply]
一样遇到这个问题

**Reply by jpuyy** [reply]
是不是要加入 wait list?
